src/maintainer/updating_pkgs.rst

```diff
@@ -32,7 +32,7 @@ When a new version of a package is released on PyPI/CRAN/.., we have a bot that
 #. Manually setting up git remotes:
 
    - Clone the conda-forge feedstock repository
-   - Add the remote of the bot: ``git add regro-cf-autotick-bot git@github.com:regro-cf-autotick-bot/<package>-feedstock.git``
+   - Add the remote of the bot: ``git remote add regro-cf-autotick-bot https://github.com/regro-cf-autotick-bot/<package>-feedstock.git``
```

Comment:
I think the URL should still be the git one

Reply:
You can't push to `git://`, only pull.

Reply:
Nvm, `git@` works.

Reply:
So should I just change it back to `git://`?

Reply:
Sorry about the confusion. It should be `git@`.

Reply:
Ok, switched it back.